app/views/layouts/_header.html.erb

```diff
@@ -12,7 +12,7 @@
           <% end %>
         </li>
         <li class="nav-item active background-slate">
-          <a class="nav-link nav-icon nav-second" href="/"><%= image_tag "BW- ADD USER ICON.svg", alt:"Add an Item" %><p>ADD ITEM</p></a></li>
+          <a class="nav-link nav-icon nav-second" href="/carriers/new"><%= image_tag "BW- ADD USER ICON.svg", alt:"Add an Item" %><p>ADD ITEM</p></a></li>
```

Comment:
Please use the rails `link_to` syntax to link to other pages. You can see a couple of lines above how that is done for the signup path. In your case, it would be a `link_to new_carrier_path ....`

There are a couple of benefits of using rails helpers like the link_to syntax:

- if you really wanna change the url, you can change it through the routes.rb file and everything will work
- if for some reason we decide to remove the ability to add new carriers, or we change the link path all together, with the rails link_to syntax, rails will raise an exception and notify us about the link not linking properly anymore, while with a href link, it will just silently fail (the link will still be there but it won't link to a correct page)